src/utils/cookies.js

```diff
@@ -0,0 +1,27 @@
+import firebase from 'firebase/app'
+import 'firebase/analytics'
+
+const hasCookiebot = () => window.Cookiebot
```

Comment:
Having Google anaytics logic within a cookie.js make it weird, I'll have either split that in two files (exporting it from the firebase.js for eg) or have an analytics.js with the cookie logic inside or something like this.